Cargo.toml

```diff
@@ -50,7 +50,7 @@ overflow-checks = false
 
 [workspace.metadata.release]
 shared-version = true
-dev-version-ext = "pre"
+dev-version-ext = "alpha.0"
```

Comment:
I changed this manually outside of `cargo release` since it was discussed to use `alpha.0` for the dev suffix, https://discord.com/channels/844353360348971068/852641819388936243/1029809792774045747

And it says beta... but my first run of this is alpha, fixing now :facepalm: